Microsoft.Toolkit.Uwp.UI.Controls/Microsoft.Toolkit.Uwp.UI.Controls.csproj

```diff
@@ -47,7 +47,7 @@
 
   <ItemGroup>
     <PackageReference Include="System.ValueTuple" Version="4.5.0" />
-	<PackageReference Include="System.Text.Json" Version="4.7.2" />
+	  <PackageReference Include="System.Text.Json" Version="4.7.2" />
```

Comment:
The edit still has a leading tab, and then two spaces, so the file is reported as mixed still.
Small fix with 4 spaces like the other lines 🙂

```suggestion
    <PackageReference Include="System.Text.Json" Version="4.7.2" />
```

Reply:
Yeah I went into vscode, guess it was not enough. Time to `"editor.renderWhitespace": "all"` I guess.

Reply:
Happy to help! 😄

And yeah enabling `renderWhitespace` in VSCode is definitely worth it too!